src/languages/en.js

```diff
@@ -1071,6 +1071,7 @@ export default {
             growlMessageOnDeleteError: 'This workspace cannot be deleted right now because reports are actively being processed',
             unavailable: 'Unavailable workspace',
             memberNotFound: 'Member not found. To invite a new member to the workspace, please use the Invite button above.',
+            notAuthorized: `You do not have access to this page. Are you trying to join the workspace? Please reach out to the owner of this workspace so they can add you as a member! Something else? Reach out to ${CONST.EMAIL.CONCIERGE}`,
```

Comment:
> Are you trying to join the workspace? Please reach out to the owner of this workspace so they can add you as a member!

@youssef-lr @sonialiap  Looking at the test steps, if the user B is already a part of the workspace, does this statement make sense?